public/res/default.json

```diff
@@ -75,7 +75,7 @@
ease note if you select a dataset, all datafiles belonging to that dataset will be selected. It is not possible for example to unselect one datafile if the parent visit is already selected. Similar to My Data, clicking on a row (not hyperlink) will display its meta data information in the panel at the bottom of the page.<h2 id='search'> Search</h2>The search tabs contains an interface the allows you search across all the visits, datasets and datafiles.<h2 id='cart'> The Cart</h2>On the very first column of each row on a grid (at the dataset or datafile level), you will see tick boxes.<br><img src='/res/images/tickbox.png'><br>Clicking this box will add or remove the selected visit to your cart. Multiple items can be added using the 'Shift' key.<br><br>You can use the Cart items counter on the top right corner to check if an item was added or removed.<br><img src='/res/images/cart-button.png'><br>Clicking on the above will open up the cart window:<br><img src='/res/images/cart.png'><br>Here you can remove already added items or request download of the items in the cart by clicking the <strong>Download Cart</strong> button. When the button is clicked, you will be presented with a window with several options.<br><img src='/res/images/cart-download.png'><br>The <strong>Download Name</strong> is a name that identifies your particular download. It is used as the filename when your download becomes ready.<br><br>The <strong>Access Method</strong> selection option determines whether you would like to download your files or whether you want them restored to another location.<br><br>You can optionally enter your email address. When your download is available, an email will be sent notifying you that your download is available.<br>Clicking the <strong>OK</strong> button on the window will submit your download request to be processed. You should see a <strong>Cart successfully submitted</strong> notification popup on the right corner. The request will be added inside the downloads window.<h2 id='download'> Download</h2>Clicking on the above will open up the downloads window:<br><img src='/res/images/downloads-button.png'><br>The Downloads window is a grid that list all your download requests.<br><img src='/res/images/downloads.png'><br>The <strong>Status</strong> column gives you the current status of your download. Possible values are <strong>Restoring from Tape</strong> and <strong>Available</strong>. The <strong>Download</strong> button is disabled when the status is restoring. Once available, you can click the download button to download the file via the browser if <strong>Https</strong> transport type was used for that request.<br><br>Please note that download pausing and resuming is not supported. Multi-part download managers are also not supported.<br><br>For downloads using Globus as the transport type, when the status is complete, you can login to <a href='https://www.globus.org/SignIn'>Globus Online</a> and initiate file transfer to your endpoint."
```

Comment:
One extra thing I did do is change TopCAT to DataGateway (it is TopCAT on develop for some reason). If there's a reason this is still TopCAT on develop I'll change it back but just wanted to point this out in case you missed it